sklearn/cluster/_agglomerative.py

```diff
@@ -999,6 +1013,13 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
 
         .. versionadded:: 0.21
 
+    compute_distances : bool, default=False
+        Computes distances even if no distance_threshold is used. This can be
+        used to make dendrogram visualization, but has impact on efficiency due
+        to the additional computational cost and used memory.
+
+        .. versionadded:: 0.24
+
```

Comment:
same comment for the `distances_` attribute.

Reply:
Fixed in 61e5c5a.